ambari-web/app/models/stack_service.js

```diff
@@ -206,7 +206,7 @@ App.StackService = DS.Model.extend({
   }.property('coSelectedServices', 'serviceName'),
 
   isHiddenOnSelectServicePage: function () {
-    var hiddenServices = ['MAPREDUCE2'];
+    var hiddenServices = [];
     return hiddenServices.contains(this.get('serviceName')) || !this.get('isInstallable') || this.get('doNotShowAndInstall');
```

Comment:
Have we made a decision to remove this? If yes, the fix would include removing the entire framework for co-selected services.

Reply:
Hi,
We have to remove hard-coded services from UI code. I will not be closing the jira after this fix so that refactoring can be done as a separate task for this jira. As for the Yarn & Mr being co-selected, it is still in discussion. cc @jayush

Reply:
@jayush @mradha25 So this means we are planning an API response for the list of co-selected services? Also, after this change along with https://github.com/mradha25/ambari/commit/28ea5b98afd08582a2eb8b5d4093b4fae8714fd5#diff-818191799511f5f6f1c55435409f4117R343 Yarn and MR will not be co-selected anymore.